content/05.discussion.md

```diff
@@ -0,0 +1,34 @@
+## Discussion
+
+We carried out a large-scale comparison of data types in the TCGA Pan-Cancer Atlas as functional readouts of genetic alterations in cancer, integrating results across cancer types and across driver genes.
+Overall, we found that gene expression and DNA methylation tend to capture signatures of mutation state equally effectively in most cases.
+We recommend focusing on these data types as downstream readouts when designing studies focused on the functional effects of cancer mutations, although results with RPPA data were also promising.
```

Comment:
This may need some more nuance. I think you might want to note that RPPA did well despite few features. Gene expression was perhaps _slightly_ better on balance, but that given the different stability of nucleic acids methylation might make more sense for some study designs. Also the caveat that we are looking primarily at the consequences of mutation as the readout.

Reply:
I tried to restructure this in 0a320dd (and moved a lot of it to a separate concluding paragraph) - let me know what you think.